frontend/assets/sass/nomad-ui.scss

```diff
@@ -58,3 +58,11 @@
 .pointer {
 	cursor: pointer;
 }
+
+dl {
```

Comment:
I don't think this will be necessary if we remove the dd/dt lists from the tables?